packages/massa-web3/src/experimental/basicElements/address.ts

```diff
@@ -18,20 +19,21 @@ const UNDERLYING_HASH_LEN = 32
  *
  * @privateRemarks
  * Interfaces are used to make the code more modular. To add a new version, you simply need to:
- * - extend the `initFromVersion` method:
- *   - Add a new case in the switch statement with the new algorithms to use.
- *   - Add a new default version matching the last version.
+ * - Add a new case in the switch statement with the new algorithms to use.
+ * - Change the DEFAULT_VERSION version matching the last version.
+ * - Change the getVersion method to detect the version from user input.
  * - check the `fromPublicKey` method to potentially adapt how an address is derived from a public key.
  * - Voila! The code will automatically handle the new version.
  */
 export class Address {
-  public bytes: Uint8Array
+  private bytes: Uint8Array
```

Comment:
Now that the subject is clarified, you should precise that this is the raw data of the address with the version but without its type.

Reply:
bytes contains address type and version. i ll add a comment